stepfunctions-local-patch/com/amazonaws/stepfunctions/local/runtime/executors/task/LambdaTaskStateExecutor.java

```diff
@@ -88,10 +91,33 @@ private AWSLambda getClient(String region) {
     }
 
     public StateResult execute() {
-        this.validateLambdaARN();
         String filteredInput = this.getFilteredInput();
         String effectiveInput = this.getEffectiveInput(this.taskState.getParameters());
 
+        // map aws-sdk tasks to lambda tasks
```

Comment:
nit: We could try adding an aspect around `getEffectiveInput(..)`, then we could potentially get rid of the source patches entirely..? (would need to look into the other/existing patches as well, though - just food for thought, maybe in a future iteration..)

Reply:
> Can we also add the source `index.ts` and a short script/README on how to generate the compiled `index.js`?

It's basically just a file that is generated by the CDK by bundling with the aws-sdk lib. Maybe it makes more sense to just use the index.js directly and package the aws-sdk in a node_modules folder in a zip file.
Still not super happy about that since the only reason we're not having a single small .js file is a shortcoming in the local executor for node.js (which doesn't provide the aws-sdk lib atm).